README.md

```diff
@@ -1,15 +1,17 @@
-# Nictiz-R4-Immunization
+# Nictiz-R4-Vaccination-Immunization
 
-This repository contains HL7 FHIR R4 compliant profiles and related conformance materials for the information standard [Immunization](https://informatiestandaarden.nictiz.nl/wiki/Immu:V0.1_Ontwerpen).
+This repository contains HL7 FHIR R4 compliant profiles and related conformance materials for the information standard [Vaccination-Immunization](https://informatiestandaarden.nictiz.nl/wiki/MedMij:V6/Ontwerp_Vaccinatie-Immunisatie).
```

Comment:
Ik geloof dat dit niet de correcte link is, de overkoepelende informatiestandaard staat wel in de Immu-namespace, alleen de MedMij-usecase staat in de MedMij-namespace. Volgens mij is dit package bedoeld voor alles binnen die overkoepelende standaard.

Reply:
Bijgewerkt naar de nieuwe landingspagina in een nieuwe commit.